CHANGES.md

```diff
@@ -1,3 +1,24 @@
+## edge-19.1.2
+
+* Controller
+  * Retry support! Introduce an `isRetryable` property to service profiles to
+    enable configuring retries on a per-route basis
+  * Add Daemonset support to Web UI, Grafana, and cli `stat` command (thanks,
+    @zknill!)
+* Web UI
+  * Add "meshed" and "no traffic" badges on the resource detail pages
+  * Fixed `linkerd dashboard` to maintain proxy connection when browser open
+    fails
+  * Fixed JavaScript bundling to avoid serving old versions after upgrade
```

Comment:
This is super nit picky, different bullet points have different tenses. "Introduce" and "Add" are present tense, but "Fixed" and "Improved" are past. I'd prefer to use the same tense throughout, but totally your call for which one you prefer.